Dockerfile

```diff
@@ -113,8 +113,8 @@ RUN rm -vf /charts/*.sh /charts/*.md
 # This image includes any host level programs that we might need. All binaries
 # must be placed in bin/ of the file image and subdirectories of bin/ will be flattened during installation.
 # This means bin/foo/bar will become bin/bar when rke2 installs this to the host
-FROM rancher/k3s:v1.21.3-k3s1 AS k3s
-FROM rancher/hardened-kubernetes:v1.21.4-rke2r4-build20210909 AS kubernetes
+FROM rancher/k3s:v1.21.5-k3s1 AS k3s
+FROM rancher/hardened-kubernetes:v1.21.5-rke2r1-build20210916 AS kubernetes
 FROM rancher/hardened-containerd:v1.4.9-k3s1-build20210908 AS containerd
 FROM rancher/hardened-crictl:v1.19.0-build20210223 AS crictl
 FROM rancher/hardened-runc:v1.0.1-build20210908 AS runc
```

Comment:
Can we bump `runc` to match upstream? 

https://github.com/kubernetes/kubernetes/blob/master/go.mod#L365

Reply:
Done